vizro-core/examples/_dev/app.py

```diff
@@ -30,7 +34,62 @@
     ],
 )
 
-dashboard = vm.Dashboard(pages=[page])
+# CREATE FAKE DATA
+column = [1, 1, 1, 1, 1, 2, 2, 2, 2, 2, 3, 3, 3, 3, 3, 4, 4, 4, 4, 4]
+row = [1, 2, 3, 4, 5, 1, 2, 3, 4, 5, 1, 2, 3, 4, 5, 1, 2, 3, 4, 5]
+value = [1, 7, 3, 5, 2, 10, 9, 1, 3, 7, 5, 8, 2, 9, 1, 2, 7, 5, 3, 2]
+group = ["A", "B", "C", "D", "E", "E", "D", "C", "B", "A", "A", "E", "C", "B", "D", "A", "D", "B", "C", "E"]
+date_time = [
+    "2023-01-01",
+    "2023-02-02",
+    "2023-03-03",
+    "2023-04-04",
+    "2023-05-05",
+    "2023-06-06",
+    "2023-07-07",
+    "2023-08-08",
+    "2023-09-09",
+    "2023-10-10",
+    "2023-11-11",
+    "2023-12-12",
+    "2024-01-01",
+    "2024-02-02",
+    "2024-03-03",
+    "2024-04-04",
+    "2024-05-05",
+    "2024-06-06",
+    "2024-07-07",
+    "2024-08-08",
+]
+
+data = pd.DataFrame()
+
+date_time_new = [datetime.strptime(date, "%Y-%m-%d").date() for date in date_time]
+
+data["COLUMN1"] = column
+data["ROW1"] = row
+data["VALUE"] = value
+data["GROUP"] = group
+data["DATE_TIME"] = date_time_new
+
+page_1 = vm.Page(
+    title="Datepicker page",
+    components=[vm.Table(id="table", figure=dash_data_table(data_frame=data))],
+    controls=[
+        vm.Filter(
+            column="DATE_TIME",
+            selector=vm.DateRangePicker(
+                title="Pick a date",
+                min_date="2023-01-01",
+                value=["2024-01-01", "2024-03-01"],
+                max_date="2024-07-07",
+            ),
+            # selector=vm.DatePicker(title="Pick a date", min_date='2023-01-01', value=['2024-01-01']),
+        )
+    ],
```

Comment:
Have you checked if DateRangePicker and DatePicker work with the vm.Parameter as well?

Reply:
Any updates on this? :)

Reply:
It works with vm.Parameter as well. :)